examples/gatsbygram/cypress/plugins/index.js

```diff
@@ -0,0 +1,17 @@
+// ***********************************************************
```

Comment:
This file was added (automatically). I can gitignore it if desired, or tie into the options cypress mentions to not create this file.